o5/dante/v1/dead_message.proto

```diff
@@ -103,7 +103,9 @@ message DeadMessageEvent {
   ];
 
   DeadMessageEventType event = 3 [
-    (psm.state.v1.event_field).event_type = true
+    (psm.state.v1.event_field).event_type = true,
+    (psm.list.v1.oneof).filtering.filterable = true,
```

Comment:
This will go on the one of itself. Like this https://github.com/pentops/protostate/blob/main/testproto/test/v1/foo.proto#L159